deployer/src/deployer/upload.py

```diff
@@ -249,6 +261,29 @@ def upload(self, bucket_manager):
             )
 
 
+class DeleteTask(UploadTask):
+    """
+    Class for doing deletion by key tasks.
+    """
+
```

Comment:
Ass mentioned in the suggestion above:
```suggestion
class DeleteTask(UploadTask):
    """
    Class for doing deletion by key tasks.
    """
    is_deletion = True
    
```